Loading parflow-ml/latest
  Loading requirement: openmpi/gcc/4.1.0 parflow/3.9.0 gdal/3.2.1
/home/SHARED/software/anaconda3/2020.07e/lib/python3.8/site-packages/setuptools/distutils_patch.py:25: UserWarning: Distutils was imported before Setuptools. This usage is discouraged and may exhibit undesirable behaviors or errors. Please use Setuptools' objects directly or at least import Setuptools first.
  warnings.warn(
Drawing 5000 posterior samples:   0%|          | 0/5000 [00:00<?, ?it/s]Drawing 5000 posterior samples: 100%|██████████| 5000/5000 [00:00<00:00, 83901.18it/s]
warning: file exists
warning: file exists
log prob true 5.003431
Traceback (most recent call last):
  File "lstm_sbi.py", line 360, in <module>
    seriesarr, fitarr = gen_Fit_Series_Wrapper(lstm_out_list, DataX_test, y_hat_full,
  File "/home/qh8373/SBI_TAYLOR/sbi_taylor/scripts/05_utils/posteriorsamputils.py", line 253, in gen_Fit_Series_Wrapper
    fitarr = gen_Fit_Wrapper()
TypeError: gen_Fit_Wrapper() missing 2 required positional arguments: 'seriesarr' and 'y_hat_full'
